google/cloud/bigquery/dbapi/_helpers.py

```diff
@@ -26,9 +26,42 @@
 
 _NUMERIC_SERVER_MIN = decimal.Decimal("-9.9999999999999999999999999999999999999E+28")
 _NUMERIC_SERVER_MAX = decimal.Decimal("9.9999999999999999999999999999999999999E+28")
+_BIGQUERY_SCALAR_TYPES = {
```

Comment:
Any chance we can use https://github.com/googleapis/python-bigquery/blob/4396e70771af6889d3242c37c5ff2e80241023a2/google/cloud/bigquery/enums.py#L219 or https://github.com/googleapis/python-bigquery/blob/4396e70771af6889d3242c37c5ff2e80241023a2/google/cloud/bigquery/enums.py#L145 instead? I worry this will get out-of-sync (already has, missing GEOGRAPHY, which just encodes as a string)

Reply:
Yes! I didn't know those existed.

So some questions:

- `SqlParameterScalarTypes` includes some aliases.  
  Is that something we want in this context? (If so, I'll lobby for `INT`. ;))

- `SqlParameterScalarTypes` and `_SQL_SCALAR_TYPES` lack `DECIMAL` and `BIGDECIMAL`. 
   https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_types
   
   Should I add `DECIMAL` and `BIGDECIMAL`?

Reply:
> SqlParameterScalarTypes and _SQL_SCALAR_TYPES lack DECIMAL and BIGDECIMAL.
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_types
> Should I add DECIMAL and BIGDECIMAL?

Good catch! Filed https://github.com/googleapis/python-bigquery/issues/633 so that we can follow-up and add those.

> SqlParameterScalarTypes includes some aliases.
> Is that something we want in this context? (If so, I'll lobby for INT. ;))

Those aliases were intended to help folks who first learned BigQuery with "legacy" syntax. https://cloud.google.com/bigquery/data-types I think it's worth allowing the same aliases here in the DB-API.

Reply:
On second-thought, I don't know if we want to be adding `DECIMAL` and `BIGDECIMAL` as aliases. Let's leave them out of this implementation for now, and we can decide later if we want more aliases beyond those we have for Legacy SQL support.

Reply:
But they're documented in the _standard_ sql docs.